cantools/database/can/formats/arxml.py

```diff
@@ -937,17 +976,36 @@ def load_string(string, strict=True):
     """Parse given ARXML format string.
 
     """
+    global NAMESPACE, NAMESPACES
```

Comment:
Is there a way to do this without globals?

Reply:
Sorry for the belated answer, for whatever reason I did not see the notification.

> Is there a way to do this without globals?

Yes. I just wanted to do this with as few changes to the code as possible. I fully agree that these variables belong into the  `SystemLoader` class, though, so I'll move them into it in the next revision of the PR. (I'm still working on it, BTW. It's just a quite a bit more involved than I originally thought.)

Reply:
Ok!

Reply:
Any progress?